src/test/kotlin/blackjack/domain/BlackJackCardMapTest.kt

```diff
@@ -0,0 +1,33 @@
+package blackjack.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+
+class BlackJackCardMapTest {
+    @Test
+    fun `카드는 에이스(1,11)부터 2~10 JACK(10) QUEEN(10) KING(10)이 있다`() {
+        val blackJackCardsSetter =
+            BlackJackCardsSetter { cards ->
+                cards.clear()
+                cards.add(BlackJackCard(BlackJackCardShape.HEART, BlackJackCardNumber.ACE))
+            }
+        val blackJackCard = BlackJackCardsMap(blackJackCardsSetter = blackJackCardsSetter).get()
+        assertThat(blackJackCard.number.values).containsExactly(1, 11)
+        assertThat(blackJackCard.shape.shape).isEqualTo("HEART")
```

Comment:
아래와 같이 equals를 테스트하면, BlackJackCardShape에 shape프로퍼티가 필요없어보이네요! 

```suggestion
        assertThat(blackJackCard.shape).isEqualTo(BlackJackCardShape.HEART)
```